icu4j/main/classes/core/src/com/ibm/icu/impl/RBBIDataWrapper.java

```diff
@@ -50,7 +51,8 @@
         /**
          * Linear array of next state values, accessed as short[state, char_class]
          */
-        public short[] fTable;
+        public short[] fTable16;
```

Comment:
I very much like Andy's suggestion, for Java only, of always deserializing the state tables into rows of 16-bit units. (I had had the same idea but didn't write it down until looking at the Java code...)

Please use `char[]` (= C++ `uint16_t[]`) instead of both `short[]` and `byte[]`.

Reply:
done